ifrs17/Import/Importers.ipynb

```diff
@@ -582,13 +582,16 @@
                 "\n",
                 "\n    if(storage.DefaultPartition != storage.TargetPartition) {",
                 "\n        var bestEstimateIvs = await workspaceToCompute.LoadPartitionedDataAsync<IfrsVariable,PartitionByReportingNodeAndPeriod>(storage.DefaultPartition);",
-                "\n        ivs = ivs.Where(iv => Math.Abs(iv.Value) >= Precision).ToArray()",
+                "\n        ivs = ivs.Where(iv => iv.Values.Any(y => Math.Abs(y) >= Precision)).ToArray()",
                 "\n            .Except(bestEstimateIvs, IfrsVariableComparer.Instance(ignoreValue: false))",
-                "\n            .Concat(ivs.Where(x => Math.Abs(x.Value) < Precision).Intersect(bestEstimateIvs, EqualityComparer<IfrsVariable>.Instance).Select(x => x with {Value = 0.0}).ToArray());",
+                "\n            .Concat(ivs.Where(iv => iv.Values.Any(y => Math.Abs(y) >= Precision)).ToArray()",
+                "\n                        .Intersect(bestEstimateIvs, EqualityComparer<IfrsVariable>.Instance)",
+                "\n                        .Select(x => x with {Values = Enumerable.Repeat(0.0d, 13).ToArray()}).ToArray());",
```

Comment:
invert the intersect and create variable with as many 0s as the length of the BE variable Values:

.Concat(bestEstimateIvs.Intersect(ivs.Where(iv =>iv => iv.Values.All(y => Math.Abs(y) < Precision))).ToArray(), EqualityComparer<IfrsVariable>.Instance)
.Select(x => x with {Values = Enumerable.Repeat(0.0d, x.Values.Length).ToArray()}).ToArray());",